recipes/bdsim-g4/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "bdsim-g4" %}
+{% set version = "1.7.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/jairhul/bdsim/get/c725b23739b8548a7a8edffb26e3b33ee49be1e7.tar.gz
+  sha256: 7c1c749d80b193bc4cc57af4bb853d65ac2bc977ba772c2ce18de76e717c9490
+
+build:
+  skip: True  # [win]
+  script: ${RECIPE_DIR}/build_bdsim.sh
+  number: 0
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - cmake
+    - make
+    - bison
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}     # [linux]
+    
+  host:
+    - clhep
+    - xerces-c    
+    - root_base >=6.32.0
+    - geant4 >=11.2.2
+    - qt-main 
+    - libzlib
+    
+  run:
+    - clhep
+    - xerces-c    
```

Comment:
All the dependencies should come in via run_exports and should not be explicitly listed.